.travis.yml

```diff
@@ -1,9 +1,9 @@
 language: rust
 sudo: false
 script:
-  - cargo build --verbose
-  - cargo test --verbose
-  - cargo doc
+  - cargo build --verbose --features="v5"
+  - cargo test --verbose --features="v5"
```

Comment:
Could you keep the original `cargo test` command as well? (just to make sure the lib works without the feature enabled)